pyshelf/bulk_update/utils.py

```diff
@@ -44,3 +46,31 @@ def run(args):
 
     container = Container(config, logger)
     container.runner.run(bucket_list)
+
+
+def run_search_prune(args):
+    """
+        Runs search cleanup based on given config file.
+
+        Args:
+            args(dict)
+    """
+    log_level = logging.INFO
+
+    if args["--verbose"]:
+        log_level = logging.DEBUG
+
+    config = {
+        "logLevel": log_level
+    }
+    configure.app_config(config, args["<config-path>"])
+
+    log_name = "prune-search-index"
+    log_file = os.path.join(config["indexPruneLogDirectory"], log_name + ".log")
+    handler = logging.FileHandler(log_file)
+    logger = logging.getLogger(log_name)
+    logger.addHandler(handler)
+    logger.setLevel(log_level)
```

Comment:
All this logging stuff is mostly duplicate (although without the format) of what is in bucket_update namespace.  Can we break that out into a different utils (maybe `pyshelf.background_utils`) and use it in both those places?